File: Presentation Tier/WebApp/Models/Ingredient.cs
namespace WebApp.Models
{
    public class Ingredient
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string UnitOfMeasure { get; set; }
        public int Quantity { get; set; }
        public double Calories { get; set; }

        public Ingredient(string name, string unitOfMeasure, double calories)
        {
            Quantity = 0;
            Name = name;
            UnitOfMeasure = unitOfMeasure;
            Calories = calories;
        }

        public override string ToString()
        {
            string s = this.Name;
            return s;
        }
    }
}

File: Data tier/Java/src/main/java/via/sep3/food/Model/Ingredient.java
package via.sep3.food.Model;


import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Ingredient {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    int Id;
    String name;
    String UnitOfMeasure;
    double Calories;


    public Ingredient() {
    }

    public Ingredient( String name, String unitOfMeasure, double calories) {
        this.name = name;
        UnitOfMeasure = unitOfMeasure;
        Calories = calories;
    }

    public String getUnitOfMeasure() {
        return UnitOfMeasure;
    }

    public void setUnitOfMeasure(String unitOfMeasure) {
        UnitOfMeasure = unitOfMeasure;
    }

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public double getCalories() {
        return Calories;
    }

    public void setCalories(double calories) {
        Calories = calories;
    }

    @Override
    public String toString() {
        return "Ingredient{" +
                "Id=" + Id +
                ", Name='" + name + '\'' +
                ", Calories=" + Calories +
                '}';
    }
}
